static/app/components/events/eventTags/util.tsx

```diff
@@ -174,7 +179,16 @@ const ISSUE_DETAILS_COLUMN_BREAKPOINTS = [
  * accurately describe the available space.
  */
 export function useIssueDetailsColumnCount(elementRef: RefObject<HTMLElement>): number {
-  const {width} = useDimensions<HTMLElement>({elementRef});
+  const [width, setWidth] = useState(0);
```

Comment:
if we're using local state we should just set the state to the number of columns